src/lib/services/email-service.ts

```diff
@@ -613,6 +634,29 @@ export class EmailService {
         throw new NotFoundError('Could not find template');
     }
 
+    private resolveTemplateAttachment(
+        templateName: string,
+        filename: string,
+        cid: string,
+    ): {
+        filename: string;
+        path: string;
+        cid: string;
+    } {
+        const topPath = path.resolve(__dirname, '../../mailtemplates');
```

Comment:
we need to check if everything works with base paths in sandbox

Reply:
this is the same as templates, but yes, we can check